pom.xml

```diff
@@ -631,31 +631,15 @@
         <artifactId>swagger-maven-plugin</artifactId>
         <version>${dep.swagger-plugin.version}</version>
         <exclusions>
-          <exclusion>
-            <groupId>org.apache.maven</groupId>
-            <artifactId>maven-plugin-api</artifactId>
```

Comment:
These deps appear to still be pulled in by the plugin, ooc why are the exclusions no longer necessary?

Reply:
so from what I've learnt the exclusion in the repo dependency management will override the exclusion in the parent-pom, ApiTooling is the only repo that uses this dependency and it only excludes `log4j` and `commons-logging`, so instead of sticking with the legacy ones here we can update to match with ApiTooling and remove the management there.